### PR TITLE
Kraken: optionally limit rate of WebSocket messages sent to server

### PR DESCRIPTION
**Pull request by eschava**
Kraken: optionally limit the rate of WebSocket messages sent to the server to avoid "Exceeded msg rate" error message

### CODE REVIEW

File: xchange-stream-kraken/src/main/java/info/bitrich/xchangestream/kraken/KrakenStreamingService.java [hunk 1]
```diff
@@ -285,4 +316,20 @@ static Integer parseOrderBookSize(Object[] args) {
     }
     return null;
   }
+
+  public static void main(String[] args)
```
Comment (earce) [inline xchange-stream-kraken/src/main/java/info/bitrich/xchangestream/kraken/KrakenStreamingService.java:320]: why are we adding a main here?

Comment (eschava) [reply xchange-stream-kraken/src/main/java/info/bitrich/xchangestream/kraken/KrakenStreamingService.java]: I'm sorry, committed a method that I used for testing. Should be OK now

**Comment by timmolter** [comment]
Please fix conflicts. I will merge and push a release.

**Comment by timmolter** [comment]
Please fix conflicts. I will merge and push a release.

**Comment by eschava** [comment]
done

**Comment by timmolter** [comment]
Sorry, it happened again. Please fix conflicts. I will merge and push a release.

**Comment by eschava** [comment]
done

**Pull request merged**